personal_assistant/classes/address_book_classes.py

```diff
@@ -31,6 +36,34 @@ def __init__(self, birthday: str):
         self.birthday = datetime.strptime(checked_birthday, "%d.%m.%Y")
 
 
+class Email(Field):
+    def __init__(self, email: str):
+        super().__init__(email)
+        self.email = self.validate_email(email)
+
+    def validate_email(self, email):
+        pattern = r"^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$"
+        if not re.match(pattern, email):
+            raise WrongEmail(
+                "Invalid email address format. Please enter the email address in the format example@example.com"
+            )
+        return email
+
+
+class Address(Field):
+    def __init__(self, address: str):
+        super().__init__(address)
+        self.address = self.validate_address(address)
+
+    def validate_address(self, address):
+        pattern = r"^\d+,\s*[\w\s]+\s*(?:street|St\.)?,\s*\w+,\s*\d{5}$"
```

Comment:
I have encountered issues when trying to enter address:
![image](https://github.com/nazarsmith/ppap7856-pythoneers-brigade-prjct/assets/61598939/fc5a8317-cf03-4288-a9e1-e3af009a2705)
I was thinking maybe we don't need this complex validation? I think something simple to make sure that the address is a string and is between 10 and 100 characters/numbers. Regex can be used for this as well
What do you think?

Reply:
the validation was fixed